dev/devicelab/pubspec.yaml

```diff
@@ -17,4 +17,4 @@ dependencies:
 
 dev_dependencies:
   # See packages/flutter_test/pubspec.yaml for why we're pinning this version.
-  test: 0.12.20
+  test: 0.12.20+12
```

Comment:
We're on `0.12.21` internally - can we bump this to that version without problems?